packages/kbn-scout/src/playwright/fixtures/parallel_run_fixtures.ts

```diff
@@ -30,7 +30,7 @@ export const scoutParallelFixtures = mergeTests(
   coreWorkerFixtures,
   scoutSpaceParallelFixture,
   // test scope fixtures
-  browserAuthParallelFixture,
+  browserAuthFixture,
```

Comment:
we don't need a custom logic here, Playwright runs workers as independent processes with own browser context. There is no issue of using the same user account concurrently in multiple browser contexts.